src/index.ts

```diff
@@ -8,13 +8,32 @@ import {
 } from 'vue'
 import { RouteRecord, RouteLocationNormalized } from 'vue-router'
 
+type Layout = { name: string; props: Record<string, unknown> }
+
+function normalizeLayout(layout: any): Layout {
```

Comment:
```suggestion
function normalizeLayout(layout: string | { name: string, props?: Record<string, unknown> }): Layout {
```

Reply:
I think it makes sense that the argument expected to be `any` as this is a "normalize" function, anything that couldn't be normalized will just simply throw an error.